screens/Home.js

```diff
@@ -120,7 +120,7 @@ const HomeScreen = () => {
           </Text>
         </AnimateChildrenIn>
 
-        <FeaturedCard />
+        <View />
```

Comment:
do you need an empty view here?

Reply:
Because it fails if we don't have it.

Reply:
It's not a good reason, but it's a real reason.